code/modules/mob/living/living_defense.dm

```diff
@@ -800,6 +801,6 @@
 
 /mob/living/proc/check_block(atom/hit_by, damage, attack_text = "the attack", attack_type = MELEE_ATTACK, armour_penetration = 0, damage_type = BRUTE)
 	if(SEND_SIGNAL(src, COMSIG_LIVING_CHECK_BLOCK, hit_by, damage, attack_text, attack_type, armour_penetration, damage_type) & SUCCESSFUL_BLOCK)
-		return TRUE
+		return SUCCESSFUL_BLOCK
```

Comment:
If you're changing this to be more bespoke about what's happening, you should also change `return FALSE` to `return NO_BLOCK` or some define (that you create)

Reply:
You also should update other uses of check block (silicons)